package/yast2-ftp-server.spec

```diff
@@ -17,7 +17,7 @@
 
 
 Name:           yast2-ftp-server
-Version:        3.2.1
+Version:        3.3.0
```

Comment:
Is it needed to change the version to 3.3.x? I mean the change is the very same as in SP3. The package content is the same, just the version differs. This will make future merges a bit more complicated due to the conflict...